src/pyorcidator/helper.py

```diff
@@ -18,15 +19,30 @@
     "Loop profile": "P2798",
     "Scopus Author ID": "P1153",
     "ResearcherID": "P1053",
+    "github": "P2037",
+    "twitter": "P2002",
+    "scopus": "P1153",
 }
-
-
-def get_external_ids(data):
-    id_list = data["person"]["external-identifiers"]["external-identifier"]
-    id_dict = {}
-    for id in id_list:
-        id_dict[id["external-id-type"]] = id["external-id-value"]
-    return id_dict
+PREFIXES = [
+    ("github", "https://github.com/"),
+    ("twitter", "https://twitter.com/"),
+    ("scopus", "https://www.scopus.com/authid/detail.uri?authorId=")
+    # TODO linkedin, figshare, researchgate, publons, semion, semantic scholar, google scholar, etc.
+]
+
+
+def get_external_ids(data) -> Mapping[str, str]:
+    """Get external identifiers that can be mapped to Wikidata properties."""
+    rv = {}
```

Comment:
what does rv mean?

Reply:
return value

Reply:
This is a questionable habit I picked up when reading through variuos Pocoo projects like click and flask

Reply:
Oh, I see, thanks! Yeah, I'd vote for us to avoid these contractions and use plain English

Reply:
Agreed, thanks for being vigilant! I will do better :)